specification/elastic/resource-manager/Microsoft.Elastic/preview/2023-02-01-preview/elastic.json

```diff
@@ -65,6 +65,52 @@
         }
       }
     },
+    "/providers/Microsoft.Elastic/listElasticVersions": {
+      "get": {
+        "tags": [
+          "ElasticVersions"
+        ],
+        "summary": "Get a list of available versions for a region.",
+        "operationId": "ElasticVersions_Get",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "query",
+            "name": "region",
+            "description": "Region where monitor resource deployment will took place.",
+            "required": true,
```

Comment:
use odata filtering instead with $filter query param. The rest api specs repo should have some examples

Reply:
This is just a regional API that fetches some elastic versions on the basis of region. Scope of this is limited to region only. I don't think we would need to use odata filtering for this at least.

Reply:
odata filters are a standard way to specify how the results need to be filtered. It does not matter whether there is one criteria or several. Can u provide a rationale for why you cannot use OData filters?